app/src/main/java/com/mathewsachin/fategrandautomata/imaging/MediaProjectionScreenshotService.kt

```diff
@@ -34,7 +34,7 @@ class MediaProjectionScreenshotService(
         val screenWidth = DisplayMetrics.widthPixels
         val screenHeight = DisplayMetrics.heightPixels
 
-        imageReader = ImageReader.newInstance(screenWidth, screenHeight, PixelFormat.RGBA_8888, 2)
+        imageReader = ImageReader.newInstance(screenWidth, screenHeight, ImageFormat.FLEX_RGBA_8888, 2)
```

Comment:
I don't see a deprecation notice for this field. Was there a Lint warning?

Reply:
The online documentation mentions some other fields as deprecated but not this one. Can't check on my laptop right now.

Reply:
The Android documentation is contradicting itself. https://developer.android.com/reference/android/media/ImageReader#newInstance(int,%20int,%20int,%20int) says that `PixelFormat` constants are okay, but then only lists a bunch of `ImageFormat` constants as possible values.

Android Studio uses that ImageFormat list for linting.

Reply:
Ok. Got it

Reply:
PixelFormat is the correct one, since ImageFormat doesn't work on my machine because of this method:

https://chromium.googlesource.com/android_tools/+/refs/heads/master/sdk/sources/android-25/android/media/ImageUtils.java#43

It says to use PixelFormat for screenshots and ImageFormat for image files.